### PR TITLE
docs: updating changelog pre release - part2

### PR DESCRIPTION
**Pull request by adisuissa**
Commit Message: updating changelog pre release - part2
Additional Description: Followup of #22194 
Risk Level: N/A
Testing: N/A
Docs Changes: N/A
Release Notes: N/A
Platform Specific Features: N/A

Signed-off-by: Adi Suissa-Peleg <adip@google.com>

### CODE REVIEW

**Comment by adisuissa** [comment]
cc @alyssawilk please LMK if you see anything strange in the latest changelog

File: changelogs/current.yaml [hunk 1]
```diff
@@ -206,6 +206,12 @@ removed_config_or_runtime:
 - area: runtime
   change: |
     removed ``envoy.restart_features.no_runtime_singleton`` and replaced with ``envoy.restart_features.remove_runtime_singleton``.
+- area: udp listener
```
Comment (alyssawilk) [inline changelogs/current.yaml:209]: added support for command operators in :ref:`TunnelingConfig hostname 

doesn't seem like a backwards incompatible change

**Review by alyssawilk — COMMENTED** [review]
it'd be nice to have consistent capitalization.  We have a  of
foo: did bar
and 
eep: Did baz


phrasing for HTTP/3 early data is a bit unusual: requires A and B to be turned on.  It's unclear they both are by default,

re-configuration -> reconfiguration

lua timeStampToString - can have docs link?

error: dubbo proxy: added metadara_path on the same line as prior router feature?

rfc1808 should have link, fixed parens

thrift changes should have links :-/  We should ping reviewer there to make sure they're done inline next time

not sure clang 14 is a new feature?

File: changelogs/current.yaml [hunk 2]
```diff
@@ -295,17 +295,14 @@ new_features:
...
-  change: |
-    official released binary is now built with Clang 14.0.0.
 - area: http
   change: |
     added :ref:`cluster_header <envoy_v3_api_field_config.route.v3.RouteAction.RequestMirrorPolicy.cluster_header>` in :ref:`request_mirror_policies <envoy_v3_api_field_config.route.v3.RouteAction.request_mirror_policies>` to allow routing shadow request to the cluster specified in the request_header.
 - area: upstream
   change: |
     added :ref:`internal upstream transport <envoy_v3_api_msg_extensions.transport_sockets.internal_upstream.v3.InternalUpstreamTransport>` for passing metadata and filter state across the user space sockets and the internal listeners.
 - area: router
-  change:
```
Comment (alyssawilk) [inline changelogs/current.yaml:302]: cc @phlax we should ideally catch this one in CI

**Review by alyssawilk — DISMISSED** [review]
Awesome!  Thanks for all the clean up!

**Comment by adisuissa** [comment]
@phlax PTAL

**Review by phlax — APPROVED** [review]
@adisuissa apologies i thought this had landed already

lgtm, not sure if you want to address @alyssawilk 's feedback further

**Comment by adisuissa** [comment]
> @adisuissa apologies i thought this had landed already
> 
> lgtm, not sure if you want to address @alyssawilk 's feedback further

All comments should be addressed.
Once this lands, and the rest of the milestone issues are done, we can start with the release process.

**Comment by phlax** [comment]
i would like half an hour or so with the final branch to run some local testing - but i can do it as we go along (#22199 )

**Pull request merged**